### ISSUE TITLE
Issues building without openssl

### ISSUE THREAD

**Issue by clevine1** [spinkham/skipfish#7]
Hello,
I am very new to Skipfish and network security tools in general, and am trying to install the tool and play around so I can prepare a class presentation on this, so I apologize if this question is obvious or easily fixable.

When I try to run the command make in order to properly build Skipfish, I run into errors with not having openssl/ssl.h:

make
cc -L/usr/local/lib/ -L/opt/local/lib src/skipfish.c -o skipfish \
        -O3 -Wno-format -Wall -funsigned-char -g -ggdb -I/usr/local/include/ -I/opt/local/include/  -DVERSION=\"2.09b\" src/http_client.c src/database.c src/crawler.c src/analysis.c src/report.c src/checks.c src/signatures.c src/auth.c -lcrypto -lssl -lidn -lz -lpcre
In file included from src/skipfish.c:41:
In file included from src/crawler.h:26:
src/http_client.h:26:10: fatal error: 'openssl/ssl.h' file not found
#include <openssl/ssl.h>
         ^~~~~~~~~~~~~~~
1 error generated.

This then runs through multiple times and ends up generating "Error 1". If it is relevant, I am on a mac with LibreSSL 2.8.3. I have looked into installing a current version of OpenSSL but there is regrettably little documentation online and I would prefer not to overwrite the current SSL version that I have. Any assistance would be much appreciated.

**Comment by TurtleWilly** [comment]
I don't think this even will build with a recent version of OpenSSL. You'll need an old version of OpenSSL.
